app/renderer/components/frame/frame.js

```diff
@@ -682,15 +683,28 @@ class Frame extends React.Component {
         isSecure = false
         const parsedUrl = urlParse(this.props.location)
         ipc.send(messages.CHECK_CERT_ERROR_ACCEPTED, parsedUrl.host, this.props.tabId)
-      } else if (['warning', 'passive-mixed-content'].includes(e.securityState)) {
+      } else if (e.securityInfo.mixedContentStatus === 'content-status-displayed') {
         // Passive mixed content should not upgrade an insecure connection to a
         // partially-secure connection. It can only downgrade a secure
         // connection.
-        isSecure = this.props.isSecure !== false ? 1 : false
+        isSecure = 1
```

Comment:
this code will actually never be reached because `securityState` is `insecure` when `e.securityInfo.mixedContentStatus === 'content-status-displayed'`. this is also an issue in 0.20.x - see https://github.com/brave/browser-laptop/issues/12742. i am fixing it

Reply:
https://github.com/brave/browser-laptop/pull/12743